utils/buildStructuredUrl.js

```diff
@@ -0,0 +1,41 @@
+/* External dependencies */
+import { isArray } from "lodash";
+
+/**
+ * Builds a URL based on a string that has parts marked with % characters, like so:
```

Comment:
I'm not sure if 'parts marked with...' is the best word choice here. I'd prefer 'variables surrounded by...'.